%YAML 1.2
---

corner_num_x : 11
corner_num_y : 8
square_size  : 0.03
